src/domain/collaboration/callout/callout.entity.ts

```diff
@@ -19,7 +19,7 @@ import { AspectTemplate } from '@domain/template/aspect-template/aspect.template
 import { CanvasTemplate } from '@domain/template/canvas-template/canvas.template.entity';
 
 @Entity()
-export class Callout extends NameableEntity implements ICallout {
+export class Callout extends NameableEntityOld implements ICallout {
```

Comment:
That name is bizarre, why do we have `NameableEntityOld`?

Reply:
This is only for the transition period until we finish with other entities (journeys, callouts and canvases). They use the nameable entity with both nameID and displayName. We could name it differently but for the transition period I'm ok with this.